microsetta_interface/implementation.py

```diff
@@ -3135,6 +3318,19 @@ def session_locale():
     return matched_locale
 
 
+def check_current_consent(account_id, source_id, consent_type):
```

Comment:
This likely should have `prerequisites`

Reply:
It's a helper function, I don't think having prerequisites would be appropriate.

Reply:
kk